odom_topic: /odom

needCheckCharger: False
charge_station_connected_topic: /charge_station_connected

sensors_topic: /sensors

navigationLaunch: "ui_package navigation.launch"
mappingLaunch: "ui_package mapping.launch"
#----------------------------------
appAddress: 192.168.88.56
portBridge: 9090
portApp: 5050
portVideo: 8090
